doc/source/getting_started/faqs.rst

```diff
@@ -0,0 +1,235 @@
+.. _faqs:
+
+Frequently asked questions
+==========================
+
+What is PyAnsys?
+----------------
+PyAnsys is a set of open source technologies that allow you to interface Pythonically
+with Ansys Fluent, Mechanical APDL, AEDT, and other Ansys products. You can use PyAnsys
+libraries within a Python environment of your choice in conjunction with external Python
+libraries.
+
+.. image:: ../_static/PyAnsys_overview.png
+  :width: 800
+  :alt: PyAnsys overview
+
+What is PyFluent?
+-----------------
+PyFluent provides Pythonic access to Ansys Fluent. Its features enable the seamless use of
+Fluent within the Python ecosystem and broad access to native Fluent features for performing
+actions such as these:
+
+- Launch Fluent using a local Ansys installation.
+- Use Fluent's TUI (text user interface) commands for both meshing and solver features.
+- Use Fluent's built-in postprocessing capabilities.
+
+PyFluent is not bundled with the Fluent installation. You must download and install PyFluent
+separately. For more information, see :ref:`faqs_install`, which appears later on this page.
+
+PyFluent has no GUI (graphical user interface). You interact with PyFluent through the Python
+environment of your choice.
+
+How does PyFluent compare to UDFs?
+----------------------------------
+PyFluent is conceptually aligned with Fluent TUI console commands (and journaling) rather than with
+UDFs (user-defined functions). In other words, PyFluent is used for automation rather than
+modifying the solver behavior.
+
+UDFs continue to be written in C and remain important elements of
+Fluent simulations.
+
+While you cannot write UDFs in Python, you can execute PyFluent commands to compile and load UDFs,
+similar to how you use TUI commands.
+
+Who should use PyFluent?
+------------------------
+Many different types of people benefit from using PyFluent to achieve various objectives. Users
+include engineers, product designers, partners, consultants, researchers, professors, students,
+and non-engineers. PyFluent is being used for multi-product automation, scientific research,
+learning and training, vertical app creation, and custom workflow integration.
+
+.. image:: ../_static/who_why_use_PyFluent.png
+  :width: 800
+  :alt: PyFluent users and objectives
+
+
+Here are some typical objectives that you can use PyFluent to achieve:
+
+- Enhance productivity with customized scripts.
+- Automate workflows with customized scripts.
+- Extend CFD simulations to a wider audience.
+- Create comprehensive workflows inspired by Python's increasingly broad offerings
+  in these areas of scientific computing:
+
+  - Computer vision
+  - ML (machine learning)
+  - AI (artificial intelligence)
```

Comment:
Are we able to demonstrate these as three distinct areas where PyFluent could be used?

Reply:
I will need a SME to address this.